qemu/tests/cfg/spapr_vty_console_login.cfg

```diff
@@ -1,5 +1,10 @@
 - spapr_vty_console_login:
     only ppc64 ppc64le
-    type = chardev_pci_serial_login
+    type = chardev_serial_login
```

Comment:
Hi zhenyzha,
Can we update ""chardev_serial_login"  to "chardev_spapr_vty_login", thanks.

Reply:
Same reason as below